couchbase-operator/templates/couchbase-bucket.yaml

```diff
@@ -9,10 +9,13 @@ items:
 - apiVersion: couchbase.com/v2
   kind: CouchbaseBucket
   metadata:
-    name: {{ default $bucket $spec.name }}
+    name: {{ default $bucket }}
     labels:
       cluster: {{ include "couchbase-cluster.clustername" $rootScope }}
   spec:
+{{- if $spec.name }}
+    name: {{ $spec.name }}
```

Comment:
This can be omitted since name is already set to spec.name (see values.yaml)

Reply:
Ah! template maps it in alphabetical order which caused me to miss it at a glance :-)